README.md

```diff
@@ -28,9 +28,8 @@ $ brew install aws-okta
 $ aws-okta add
 ```
 
-This will prompt you for your Okta organization, username, and password.  These credentials will then be stored in your keyring for future use.   
-Note: Don't append ".okta.com" to your organization name, it is automatically added by `aws-okta`.
-
+This will prompt you for your Okta organization, custom domain, region, username, and password. These credentials will then be stored in your keyring for future use. 
+Note: please use custom domain to remote .okta.com domain
```

Comment:
I think you meant `remove`? Even then I'm not sure the note is necessary. If you're a new user reading the README, you won't have enough context to make sense of this. If you've already used it, the UI makes it apparent I think